### DISCUSSION TITLE
Wrong Chinese Translation on Explore Feed

### DISCUSSION THREAD

**Discussion by tzhengus** [community/community#123593] [Mobile]
It seems like in the Simplified version of the mobile app, under explorer. The translation of "X stared a repository" is translated to something like “X个带有星标的1个仓库” which does not make any sense.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.161.0 (130088629)
Device: iPhone 15 Pro Max
</details>

**Comment by aconsuegra** [comment]
Hi @dgzzhb, thanks a lot for the report! Do you have any suggestion about how to make this translation better?
